pages/teachers/djacobs/giza.tsx

```diff
@@ -74,6 +81,50 @@ export default function djacobs(props) {
     console.log(guess);
   };
 
+  const [insertGizaStudentData, insertGizaStudentDataMutation] = useMutation(
+    ADD_GIZA_DATA,
+    {
+      onCompleted: (data) => {
```

Comment:
You're not even doing anything here besides logging. Just remove the function

Reply:
Resolved in latest commit